src/Core/include/OpenLoco/Core/Timer.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <chrono>
+#include <cstdint>
+
+namespace OpenLoco::Core
+{
+    class Timer
```

Comment:
Are you able to add some tests for this class?

Reply:
What kind of test? I can only test if the duration is somewhat close to what we would expect.

Reply:
I dunno just a simple little thing so that there is an easy to reference example for use.